.github/workflows/scripts/validate/index.js

```diff
@@ -0,0 +1,57 @@
+const Schema = require('validate')
+const Toml = require('toml')
+
+const testIssue = `
+id = "projectriff/node-function"
+version = "0.6.2"
+addr = "gcr.io/projectriff/node-function@sha256:9d88250dfd77dbf5a535f1358c6a05dc2c0d3a22defbdcd72bb8f5e24b84e21d"
+`
+const bodySchema = new Schema({
+    id: {
+        type: String,
+        required: true,
+    },
+    version: {
+        type: String,
+        match: /^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$/,
```

Comment:
can we use a Semver matching library instead? This is a very impressive, and very intimidating regex

Reply:
maybe, but I'd prefer not to add another library if not necessary.  I actually didn't come up with regex.  I took it off the semver official [website](https://semver.org/).